var/spack/repos/builtin/packages/qt/package.py

```diff
@@ -173,6 +176,7 @@ class Qt(Package):
     depends_on("libpng", when='@4:')
     depends_on("dbus", when='@4:+dbus')
     depends_on("gl", when='@4:+opengl')
+    depends_on("assimp@5.0", when='@5.14:+opengl')
```

Comment:
```suggestion
    depends_on("assimp@5.0.0:5.0", when='@5.14:+opengl')
```
I don't think there are any plans to have another assimp 5.0.x release.

Reply:
Yes, but I'm assuming that since spack has only 5.0.1 (not 5.0, not 5.0.0, not 5.0.2) that its recipe is incomplete and we could have a new assimp available?

Reply:
I haven't quite figured out spack's policy on retroactive version completeness. If there is a 5.0.1, then do we need to add support for the 5.0.0 version? I personally think those gaps are fine, especially for smaller packages and especially for previous bugfix releases (since invariably some of these older versions will need some patches, which increases support burden).